config/routes.rb

```diff
@@ -1,7 +1,7 @@
 Rails.application.routes.draw do
   get 'home/index'
   # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
-  resource :articles
+  resources :articles
```

Comment:
This will set you up with routes pointing to all controller endpoints: `[:new, :edit, :create, :update, :destroy, :index, :show]`. Looks like you've not implemented a couple of them, so you can exclude them like this:

`resources :articles, except: [:edit, :update, :index]`

[Read more about `resources`](http://guides.rubyonrails.org/routing.html#restricting-the-routes-created)

Then later, when or if they're implemented, remove those actions from the exception list.

Reply:
Gotcha thanks!!